src/util.cpp

```diff
@@ -1338,14 +1338,14 @@ int64_t GetAdjustedTime()
     return GetTime() + GetTimeOffset();
 }
 
-bool IsLockTimeWithin14days(int64_t locktime)
+bool IsLockTimeWithin14days(int64_t locktime, int64_t reference)
 {
-    return IsLockTimeWithinMinutes(locktime, 14 * 24 * 60);
+    return IsLockTimeWithinMinutes(locktime, reference, 14 * 24 * 60);
 }
 
-bool IsLockTimeWithinMinutes(int64_t locktime, int minutes)
+bool IsLockTimeWithinMinutes(int64_t locktime, int64_t reference, int minutes)
```

Comment:
Woah! Is the order of parameters right? This is easily to get wrong.
Looks correct on where I looked. This function should be deprecated however.

Reply:
Hmm, you might be right. The argument order does not match the order in the function name.